app/src/main/java/com/door43/translationstudio/core/ImportUsfm.java

```diff
@@ -1087,11 +1087,14 @@ private boolean breakUpChapter(CharSequence text, String currentChapterStr) {
                 return false;
             }
         } else { // save stuff before first chapter
-            String chapter0 = "00"; // chapter "00" folder contains stuff that applies to the whole book, like title
-            success = saveSection("front", "intro", cleanedString);
-            successOverall = successOverall && success;
-            success = saveSection(chapter0, "title", mBookName);
-            successOverall = successOverall && success;
+            String strippedUsfmFrontTags = removeKnownUsfmTags(cleanedString);
+            if(strippedUsfmFrontTags.length() > 0) {
+                String chapter0 = "00"; // chapter "00" folder contains stuff that applies to the whole book, like title
+                success = saveSection("front", "intro", strippedUsfmFrontTags);
+                successOverall = successOverall && success;
+                success = saveSection(chapter0, "title", mBookName);
```

Comment:
fyi. you might want to check in the `front` dir for the `title` first and then fall back to checking the `00` dir. Since the `00` dir is now legacy.